docs/setup_guide.md

```diff
@@ -52,17 +52,30 @@ For this guide we will assume that the returned template id is `0x00000000000000
 
 ### Deploying the module
 
-Now that we have a template, a hardhat task can be used to deploy a DAO module instance. This setup task requires the following parameters: `dao` (the address of the Safe), `oracle` (the address of the Realitio contract) and `template` (the template to be used with Realitio). There are also optional parameters, for more information run `yarn hardhat setup --help`.
+Hardhat tasks can be used to deploy a dao module instance. There are two different tasks, the first one is through a normal deployment and passing arguments to the constructor (with the task `setup`), or, deploy the Module through a [Minimal Proxy Factory](https://eips.ethereum.org/EIPS/eip-1167) and save on gas costs (with the task `factorySetup`) - In rinkeby the address of the Proxy Factory is: `0xd067410a85ffC8C55f7245DE4BfE16C95329D232` and the master copy of the DAO Module: `0x4D0D4Bd6eCA52f2F931c099B6a8a8B2ae85FFD4E`.
+
+Now that we have a template, a hardhat task can be used to deploy a DAO module instance. These tasks requires the following parameters:
+
+- `executor` - the address of the executor.
+- `owner` - the address of the owner
+- `oracle` - the address of the Realitio contract
+- `template` - the template to be used with Realitio
+
+There are also optional parameters, for more information run `yarn hardhat setup --help` or `yarn hardhat factory-setup --help`.
```

Comment:
Wouldn't it be nicer to use a flag for the task instead of 2 tasks (e.g. `--proxied`)

Reply:
100%. I will update the code accordingly